### ISSUE TITLE
Tables and latex symbols don't appear correctly

### ISSUE THREAD

**Issue by Kiguli** [arXiv/html_feedback#482]
### Description

Table doesn't display correctly, also symbols like hat over the variables appears quite weirdly.

### (Required) What is this issue most closely related to? Select one.

Choose One

### Internal issue ID

138a666d-bfa1-4603-badc-c86da1dcab7d

### Paper URL

https://arxiv.org/html/2401.03555v1

### Browser

Chrome/121.0.0.0

### Device Type

Desktop

**Comment by Kiguli** [comment]
Here is a screenshot:

![image](https://github.com/arXiv/html_feedback/assets/38587669/ba1f1bc4-5fd9-4164-a8c8-8f131b9ab168)

**Comment by chrisjcameron** [comment]
The table is rotated and the caption is cut off. I assume the rotation is intentional and the cutoff is the issue?

The hat display is partially browser-dependent and Chrome has the most issues.

**Comment by Kiguli** [comment]
Yes, the rotation is intentional as it could fit on one page landscape but was too wide for the page in portrait, the issue is that the table is now only partially visible. Thanks

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.
